src/OrbitGl/TimelineUi.h

```diff
@@ -11,6 +11,13 @@
 
 namespace orbit_gl {
 
+namespace {
+
+const Color kMajorTickColor(255, 254, 253, 255);
+const Color kMinorTickColor(255, 254, 253, 63);
+
+}  // namespace
```

Comment:
Since this is defined in a header you want to use `inline` instead of an anonymous namespace.